test/entry/entry-single-arg.test.js

```diff
@@ -10,7 +10,9 @@ describe('single entry flag', () => {
         const { stdout, stderr } = run(__dirname);
         const summary = extractSummary(stdout);
         const outputDir = 'entry/bin';
-        expect(summary['Output Directory']).toContain(outputDir);
+        const outDirectoryFromCompiler = summary['Output Directory'].split(sep);
+        const outDirToMatch = outDirectoryFromCompiler.slice(outDirectoryFromCompiler.length - 2, outDirectoryFromCompiler.length).join('/');
+        expect(outDirToMatch).toContain(outputDir);
         expect(stderr).toContain('Entry module not found');
         stat(resolve(__dirname, './bin'), (err, stats) => {
```

Comment:
`./bin` is not correct. It should be just `bin`

Reply:
done